Source/santactl/Commands/SNTCommandFileInfo.m

```diff
@@ -305,13 +310,13 @@ - (SNTAttributeBlock)rule {
           output = @"None".mutableCopy;
           break;
       }
-      if ((SNTEventStateAllow & s) && isatty(STDOUT_FILENO) && !fi.jsonOutput) {
+      if ((SNTEventStateAllow & s) && PrettyOutput()) {
```

Comment:
I think the "PrettyOutput" condition should be on its own to avoid repeating it:

```objc
if (PrettyOutput()) {
  if (s & SNTEventStateAllow) {
    ...
  } else if, etc. etc.
```